crates/relayer/src/connection.rs

```diff
@@ -930,9 +931,12 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Connection<ChainA, ChainB> {
         &self,
         consensus_height: Height,
     ) -> Result<(), ConnectionError> {
-        // skip if we do not check the consensus proof height
-        if consensus_height.revision_height() == u64::MAX {
-            return Ok(());
+        // FIXME: no need to check proof height for now
```

Comment:
When do we need to check it?

Reply:
I'm not sure now, many until we work for the relaying between Axon and Cosmos